src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -269,6 +274,7 @@ static bool addTriggeredSubscriptions
       subs.insert(std::pair<string, TriggeredSubscription*>(subIdStr, trigs));
     }
   }
+  releaseMongoConnection(connection);
```

Comment:
Anything we can do to 'reset' the cursor
If there is anything more that we can do to clean the cursor for the next user, let's do it ...